CMakeLists.txt

```
@@ -69,9 +75,9 @@ from . import contrib\n\
 \")")
 
 # Recurse
-add_subdirectory("vgl" "vgl-build")
-add_subdirectory("vil" "vil-build")
-add_subdirectory("vnl" "vnl-build")
-add_subdirectory("vpgl" "vpgl-build")
-add_subdirectory("contrib" "contrib-build")
+add_subdirectory("vgl" "pyvxl_build/vgl-build")
+add_subdirectory("vil" "pyvxl_build/vil-build")
+add_subdirectory("vnl" "pyvxl_build/vnl-build")
+add_subdirectory("vpgl" "pyvxl_build/vpgl-build")
+add_subdirectory("contrib" "pyvxl_build/contrib-build")
```

@NoahRJohnson Why did you add pyvxl_build here? In the [past](https://bitbucket.org/visionsystemsinc/core3d/src/c149d25a3195ca73ee4ec71b55536c24f1df2b77/Core3D/CMakeLists.txt#lines-30) we used `CMAKE_BINARY_DIR`, so 
```suggestion
add_subdirectory("contrib" "${CMAKE_BINARY_DIR}/pyvxl-contrib-build")
```

[Reference](https://www.johnlamp.net/cmake-tutorial-4-libraries-and-subdirectories.html#ToDoCore/unit_test/CMakeLists.txt-13)

---

The idea was to congregate all of the different pyvxl submodule build paths under a single pyvxl build, so that the parent project's Build dir doesn't get cluttered. The paths are relative to the current output directory, but I can make them absolute by prepending ${CMAKE_BINARY_DIR} to them. This would leave us with paths like:
add_subdirectory("contrib" "${CMAKE_BINARY_DIR}/pyvxl_build/contrib-build")